doc/Porting.md

```diff
@@ -0,0 +1,94 @@
+# Porting
+
+This page aims to provide guidance on how to port KOReader to other platforms.
+
+There are mainly two modules that you need to take care of: input and output.
+After you finish these two, KOReader should have no problem running on your
+platform. Feel free to open issues in our issue tracker if you need further help on this topic :)
+
+
+## Output Module
+
+KOReader uses framebuffer to control EInk devices, so the output module here is
+[einkfb.c][einkfb-c]. You can find this file in koreader-base framework. The
```

Comment:
This section is out of date now. einkfb has been ported to lua under `base/ffi/framebuffer_einkfb.lua`.